src/components/Sidebar/Room.module.scss

```diff
@@ -1,27 +1,27 @@
 .roomSelector {
-    margin: 8px;
-    text-align: left;
-    padding: 20px;
-    font-size: 18px;
-    font-family: "Karelia";
-    background-color: rgb(255, 255, 255);
-    border: none;
-    height: 92px;
-    width: 350px;
-    border-radius: 20px;
+  margin: 8px;
+  text-align: left;
+  padding: 20px;
+  font-size: 18px;
+  font-family: 'Karelia';
```

Comment:
@mxinburritos should this have any fallback fonts or is it handled somewhere else?